ush/nems_configure.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+
+#####
+## This script writes nems.configure file 
+## first, select a "*.IN" templates based on 
+## $confignamevarfornems and parse values based on 
+## $cpl** switches.
+##
+## This is a child script of modular
+## forecast script. This script is definition only
+#####
+writing_nems_configure()
+{
+echo "SUB ${FUNCNAME[0]}: parsing_nems_configure begins"
+if [ -e $SCRIPTDIR/nems.configure ]; then
+	rm -f $SCRIPTDIR/nems.configure
+fi
+
+# Setup nems.configure
+DumpFields=${NEMSDumpFields:-false}
+cap_dbug_flag=${cap_dbug_flag:-0}          
+if [ $warm_start = ".true." ]; then
+  cmeps_run_type='continue'
+else 
+  cmeps_run_type='startup'
+fi
+if [[ $inistep = "cold" ]]; then
+  restart_interval=0
+  coldstart=true     # this is the correct setting
+else
+  restart_interval=${restart_interval:-3024000}    # Interval in seconds to write restarts
+  coldstart=false
+fi
+
+ATM_model=${ATM_model:-'fv3'}
+OCN_model=${OCN_model:-'mom6'}
+ICE_model=${ICE_model:-'cice'}
+WAV_model=${WAV_model:-'ww3'}
+CHM_model=${CHM_model:-'gocart'}
+
+ATMPETS=${ATMPETS:-8}
+MEDPETS=${MEDPETS:-8} 
+OCNPETS=${OCNPETS:-8}
+ICEPETS=${ICEPETS:-8}
+WAVPETS=${WAVPETS:-8}
+CHMPETS=${CHMPETS:-${ATMPETS}}
+
+USE_MOMMESH=${USE_MOMMESH:-"true"}
+
+
+rm -f $DATA/nems.configure
+
+med_petlist_bounds=${med_petlist_bounds:-"0 $(( $MEDPETS-1 ))"}
+atm_petlist_bounds=${atm_petlist_bounds:-"0 $(( $ATMPETS-1 ))"} 
+ocn_petlist_bounds=${ocn_petlist_bounds:-"$ATMPETS $(( $ATMPETS+$OCNPETS-1 ))"}  
+ice_petlist_bounds=${ice_petlist_bounds:-"$(( $ATMPETS+$OCNPETS )) $(( $ATMPETS+$OCNPETS+$ICEPETS-1 ))"} 
+wav_petlist_bounds=${wav_petlist_bounds:-"$(( $ATMPETS+$OCNPETS+$ICEPETS )) $(( $ATMPETS+$OCNPETS+$ICEPETS+$WAVPETS-1 ))"} 
+chm_petlist_bounds=${chm_petlist_bounds:-"0 $(( $CHMPETS-1 ))"}
+
+# Copy the selected template into run directory
+cp $SCRIPTDIR/nems.configure.$confignamevarfornems.IN tmp1
+sed -i -e "s;@\[med_model\];cmeps;g" tmp1
+sed -i -e "s;@\[atm_model\];$ATM_model;g" tmp1
+sed -i -e "s;@\[med_petlist_bounds\];$med_petlist_bounds;g" tmp1
+sed -i -e "s;@\[atm_petlist_bounds\];$atm_petlist_bounds;g" tmp1
+
+if [ $cplflx = .true. ]; then
+    if [ $restart_interval  -gt 0 ]; then 
+      restart_interval_nems=$restart_interval 
+    else 
+      restart_interval_nems=$FHMAX 
+    fi
+        sed -i -e "s;@\[ocn_model\];$OCN_model;g" tmp1
```

Comment:
Indentation of these sed lines should be uniform

Reply:
Resolved